scripts/run_test_plan.sh

```diff
@@ -240,6 +240,8 @@ JVM_ARGS="-Xms4g -Xmx64g -Xss250k -XX:MaxMetaspaceSize=1g" $JMETER_HOME/bin/jmet
     -JK8S_BEARER_TOKEN=$K8S_BEARER_TOKEN \
     -JHTTP_SERVER=$HTTP_SERVER \
     -JHTTP_SERVER_PORT=$HTTP_SERVER_PORT \
+    -JTEST_DIR=$test_dir \
+    -JSCRIPTS_DIR=`pwd` \
```

Comment:
pls replace backquotes with `$(pwd)`

Reply:
also, if the test is being executed from an external folder, than this will not work as expected.
in that case we should either document it or find the path relative to the invoked script.